fdbserver/LogSystemDiskQueueAdapter.h

```diff
@@ -127,7 +127,7 @@ class LogSystemDiskQueueAdapter final : public IDiskQueue {
 	// Recovery state (used while readNext() is being called repeatedly)
 	bool enableRecovery;
 	Reference<ILogSystem> logSystem;
-	Version recoveryLoc, recoveryQueueLoc;
+	Version startLoc, recoveryLoc, recoveryQueueLoc;
```

Comment:
A comment about what these versions are about (or, at least, what "startLoc" is about) would be nice. Thanks!